### ISSUE TITLE
5-card major bid before longer minor

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#106]
0FFD9314E5A8CB920FC58D291DDA
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/66e227a6-7272-4e09-99e8-288d2210f41b)

In my opinion this is bad bidding.

With gameforcing values you should always bid the longest suit.

With weaker hands you bid the major before the longer minor if you have to go a level higher to bid the minor.

BTW, Jump in 4th suit is splinter like you write, but it seems WBridge5 plays it as natural.
To show a natural suit, responder will have to rebid the 4th suit.

**Comment by EdwardPiwowar** [comment]
This is true in theory, but in practice it is difficult to bid. For now, BBA prefers major fives.
You gave better examples of bidding for one of the GIBs, the other GIB couldn't handle those hands.
